src/screens/shared/auth/SignUp.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+
+import { Button } from '@material-ui/core';
+
+import Dropdown from '@components/Dropdown';
+import FieldInput from '@components/FieldInput';
+import '@styles/SignUp.css';
+import { validateField, updateUserFields } from '@lib/utils';
+
+class SignUp extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: '',
+      userTypes: '',
```

Comment:
`userTypesIndex` should be initialized at 1 or 0 or '', 
![image](https://user-images.githubusercontent.com/35586281/114112976-6f9adf80-9892-11eb-9d31-0b5c06fe1ee8.png)